File: SistemaParadaCerta/APIs-ParadaCerta/paradacerta-api/src/main/java/com/paradacerta/api/model/ReservaResponse.java
package com.paradacerta.api.model;

import lombok.AllArgsConstructor;
import lombok.Data;

import java.math.BigDecimal;

@Data
@AllArgsConstructor
public class ReservaResponse {
    private String sessaoId;
    private Integer estacionamentoId;
    private String estacionamentoNome;
    private String pixKey;
    private long horaEntrada;       // Unix timestamp em ms
    private long inicioReservaPrevisto; // Unix timestamp em ms
    private BigDecimal precoHora;
    private String placa;
    private String modeloVeiculo;
}

File: SistemaParadaCerta/AplicacaoMobile-ParadaCerta/ParadaCerta/app/src/main/java/com/example/paradacerta/models/ReservaResponse.kt
package com.example.paradacerta.models

data class ReservaResponse(
    val sessaoId: String,
    val estacionamentoId: Int,
    val estacionamentoNome: String,
    val pixKey: String?,
    val horaEntrada: Long,
    val inicioReservaPrevisto: Long?,
    val precoHora: Double,
    val placa: String?,
    val modeloVeiculo: String?,
    val status: String? = null
)
